### PR TITLE
 add drunken peanut butter cups recipe

### PR DESCRIPTION
**Pull request by cvick**
Add the recipe for drunken peanut butter cups and add it to the featured Halloween drinks list 🎃

### CODE REVIEW

File: src/recipes/drunken-peanut-butter-cups.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            "ingredient": "chocolate syrup"
+        }
+    ],
+    "directions": [
+        "In a sealable container (for example a mason jar), combine vodka and Reese's Pieces. Cover with a tight-fitting lid and shake. Refridgerate overnight, then strain.",
+        "Fill a cocktail shaker with ice, then pour over strained vodka, milk, and chocolate liqueur. Shake until chilled.",
+        "Put peanut butter on one small plate and crushed Reese's Pieces on another. Dunk tops of two cocktail glasses in peanut butter then roll in crushed Reese's Pieces.",
+        "Drizzle about a tablespoon of chocolate syrup into each glass. Divide cocktail mixture between glasses and garnish each with a peanut butter cup."
+    ],
+    "image": "<drunken-peanut-butter-cups.jpg>",
```
Comment (alfg) [inline src/recipes/drunken-peanut-butter-cups.json]:
```suggestion
    "image": "drunken-peanut-butter-cups.jpg",
```

Comment (cvick) [reply src/recipes/drunken-peanut-butter-cups.json]: Thanks, can't believe I forgot to remove those!

**Comment by cvick** [comment]
> Hi, sorry one more request. If it's possible to use an image 600x400px (horizontal)?

No problem, just pushed a commit with the proper dimensions!

**Comment by alfg** [comment]
Thanks!

**Pull request merged**